application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -54,12 +59,29 @@ private boolean messageHasNoMediaAttached(Message message) {
                 && !message.getContentRaw().contains("http");
     }
 
-    private RestAction<Message> dmUser(Message message) {
+    private MessageEmbed originalMessageEmbed(Message message) {
         String originalMessageContent = message.getContentRaw();
-        MessageEmbed originalMessageEmbed =
-                new EmbedBuilder().setDescription(originalMessageContent)
-                    .setColor(Color.ORANGE)
-                    .build();
+        return new EmbedBuilder().setDescription(originalMessageContent)
+            .setColor(Color.ORANGE)
+            .build();
+    }
+
+    private RestAction<Message> pingUserAobutMedia(Message message) {
+        MessageEmbed originalMessageEmbed = originalMessageEmbed(message);
+
+        long authorId = message.getAuthor().getIdLong();
+
+        MessageCreateData pingMessage = new MessageCreateBuilder().setContent("Hey there, you <@"
+                + authorId
+                + "> posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
+            .setEmbeds(originalMessageEmbed)
+            .build();
```

Comment:
* dont mention manually, instead use JDAs methods, like `author.getAsMention()`.
* lots of code duplication with the `dmUser` method. instead, let both methods use the same text and just add the ping as prefix or similar here.

Reply:
What is the duplicate I expected both to have different message in future so I did make them different

Reply:
they are the same just with a ping in between(which can go anywhere)

Reply:
ok I will change it